--- trunk/Sambuca Chicken.txt
+++ trunk/Sambuca Chicken.txt
@@ -1,8 +1,17 @@
 Ingredients:
-chicken
-whipping cream
-Sambuca liqueur
-orange juice concentrate
+1/4 cup butter
+2 whole boneless, skinless chicken breasts, cubed
+2 garlic cloves, minced
+3 cups (750 ml) whipping cream
+1/4 cup Sambuca liqueur
+1/4 cup orange juice concentrate
+1/2 cup freshly grated Parmesan cheese
+1/4 tsp. nutmeg
+salt & pepper to taste
+1 - 14 oz. can (398 ml) artichokes, drained & chopped
+3 oz. pkg. (85 g) dried cranberries
+1 lb. fettuccine
+chopped fresh parsley & orange zest for garnish
 
 Instructions:
-In a large frying pan, saute chicken until barely cooked. Add whipping cream, Sambuca and orange juice to pan and stir well.
+In a large frying pan, heat butter and saute cubed chicken and garlic until barely cooked (do not over cook!). Remove chicken from pan and set aside. Add whipping cream, Sambuca and orange juice to pan and stir well. Add Parmesan, nutmeg, salt and pepper, stirring until smooth. Simmer for 5 minutes. Add chopped artichokes, cranberries and chicken; simmer and stir for 10 minutes. That's it for the sauce! Now, cook the fettuccine according to package directions. Carefully fold sauce into pasta and garnish with parsley and orange zest. Serve with garlic toast. Serves 6-8
