--- Sambuca Chicken.txt
+++ Sambuca Chicken.txt
@@ -1,8 +1,8 @@
 Ingredients:
-chicken
+beef
 whipping cream
 Sambuca liqueur
 orange juice concentrate
 
 Instructions:
-In a large frying pan, saute chicken until barely cooked. Add whipping cream, Sambuca and orange juice to pan and stir well.
+In a large frying pan, saute beef until barely cooked. Add whipping cream, Sambuca and orange juice to pan and stir well.
